LifeCycleAssessment_oM/Results/ElementResult.cs

```diff
@@ -0,0 +1,93 @@
+﻿using BH.oM.Analytical.Results;
+using BH.oM.Base;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace BH.oM.LifeCycleAssessment.Results
+{
+    [Description("Result for a single element from an LCA assesment. Contains total value as well as breakdown per material/EPD evaluated.")]
+    public class ElementResult : IResult, IObjectResult, IResultItem, IImmutable
+    {
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("Id of the BHoMObject that this result belongs to.")]
+        public virtual IComparable ObjectId { get; } = "";
+
+        [Description("Scope the object this result was generated from belongs to, e.g. Foundation or Facade")]
+        public virtual ScopeType Scope { get; } = ScopeType.Undefined;
+
+        [Description("Category of the object this result was generated from, e.g. Beam or Wall")]
+        public virtual ObjectCategory Category { get; } = ObjectCategory.Undefined;
+
+        [Description("Phase of life abbreviation for the scope of the EPD. A single EnvironmentalMetric can contain either a single Phase or a list of Phases i.e. A1, A2, A3.")]
+        public virtual IReadOnlyList<LifeCycleAssessmentPhases> Phases { get; } = new List<LifeCycleAssessmentPhases>();
+
+        [Description("The total quantity of the specified Field within the object.")]
+        public virtual double Quantity { get; set; }
+
+        [Description("The EPD Field selected for evaluation.")]
```

Comment:
More verbose: Environmental Product Declaration

Reply:
```suggestion
        [Description("The Environmental Product Declaration Field selected for evaluation.")]
```

Conversation was marked as resolved @IsakNaslundBh so not sure if it's just been missed from a push perhaps?

Reply:
You are right. Updated it in one place but not the other